cluster-up/cluster/kind/common.sh

```diff
@@ -71,6 +71,10 @@ function _configure_registry_on_node() {
     ${NODE_CMD} $1  sh -c "echo $(docker inspect --format '{{.NetworkSettings.IPAddress }}' $REGISTRY_NAME)'\t'registry >> /etc/hosts"
 }
 
+function _port() {
+	docker inspect "${CLUSTER_NAME}"-"$@" --format='{{range $p, $conf := .NetworkSettings.Ports}}{{(index $conf 0).HostPort}}{{end}}'
```

Comment:
Since this function accepts a single mandatory parameter, can you please assign it to a local variable at the beginning of the function? Also, you probably should verify that the variable that this function needs to operate properly was indeed passed or fail if it wasn't.

Something like:

```
function _port() {
	local container_name="${1:?}"
	...
}
```

This will make the function more readable and easier to use by others.